ui/tutorial-renderer/src/main/res/layout/tutorial_tips_start.xml

```diff
@@ -5,69 +5,19 @@
         <variable name="callbacks" type="org.cru.godtools.tutorial.TutorialCallbacks" />
     </data>
 
-    <androidx.constraintlayout.widget.ConstraintLayout
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:clipToPadding="false"
-        android:paddingTop="@dimen/tutorial_page_inset_top"
-        android:paddingBottom="@dimen/tutorial_page_inset_bottom">
```

Comment:
@JasonHammar this paddingTop and paddingBottom would need to be present on the column as a whole